test/files/rules/semicolon.test.ts

```diff
@@ -36,3 +36,13 @@ function useStrictMissingSemicolon() {
     "use strict"
     return null;
 }
+
+class MyClass {
+    public name : string
+    private index : number
```

Comment:
please add a member with _correct_ semicolon placement to `MyClass` and `ITest` so that we can test for false positives.